cvat-ui/src/reducers/import-reducer.ts

```diff
@@ -36,7 +33,12 @@ const defaultState: ImportState = {
     tasks: {
         dataset: {
             modalInstance: null,
-            current: {},
+            uploadState: {
+                id: null,
+                format: '',
+                progress: 0,
+                status: '',
+            },
```

Comment:
If it was supposed to add upload progress bar (like we have when importing a dataset to a project) to tasks and jobs
It does not work. If not, please, explain

Reply:
Removed. That field is used only for projects